### DISCUSSION TITLE
Profiler Enhancements for 11ax

### DISCUSSION THREAD

**Discussion by wifinigel** [WLAN-Pi/wlanpi-profiler#105] [Ideas]
Profiler is a very powerful, popular feature of the WAN Pi toolset.

We are currently looking at a number of enhancements to improve the scope of its reporting to include 802.11ax feature support.

It would be interesting to hear which 802.11 features (11ax or otherwise) you'd like to see profiler reporting.

We extract client capability information from association request frames, so any features you'd like to see included need to be available within the information elements of these frames.

Thanks!

Nigel.

**Comment by Devinator** [comment]
When I choose the no-11r and no-11ax options in the WLANpi Pro, it doesn't seem to disable those features in the beacons/probes.
![Profiler](https://user-images.githubusercontent.com/7076138/156632532-489de7eb-1698-4330-a539-cb9688dfd0bd.jpg)

**Reply by adriangranados** [reply]
This issue has been resolved. Please login to the Pi and run "sudo apt-get update", "sudo apt-get upgrade" to install the latest fixes.

**Reply by Devinator** [reply]
Thanks Adrian!
